src/functions.php

```diff
@@ -83,16 +81,17 @@ function getFtpConnection($url, $user, $password, $directory, $secure)
 /**
  * upload image files via ftp to the fritzbox fonpix directory
  *
- * @param stdClass[] $vcards downloaded vCards
+ * @param object $vcards downloaded vCards
```

Comment:
Should be `mixed[]` or array?

Reply:
ping

Reply:
build back a few commits ago -> outdated